### PR TITLE
Los Angeles County Infant Supplement and Expectant Parent Payment

### LINKED ISSUE #5319: Los Angeles County Infant Supplement and Expectant Parent Payment

### PR DESCRIPTION
**Pull request by PavelMakarchuk**
Fixes #5319

### CODE REVIEW

**Review by MaxGhenis — CHANGES_REQUESTED** [review]
Shouldn't we make all these monthly?

Could you cite the WIC (legal code) pointed to in the pdf doc? In particular I'm unclear on the definition of infant here.

Also check some typos in descriptions

**Comment by PavelMakarchuk** [comment]
> Shouldn't we make all these monthly?
> 
> Could you cite the WIC (legal code) pointed to in the pdf doc? In particular I'm unclear on the definition of infant here.
> 
> Also check some typos in descriptions

Refactored to monthly and added the infant definition

File: policyengine_us/parameters/gov/local/ca/la/dss/expectant_parent_payment/amount.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+description: The California Department of Social Services provides the following Expectant Parent Payment amount, based on pregnancy months.
```
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/local/ca/la/dss/expectant_parent_payment/amount.yaml:1]: `based on` usually for breakdowns
```suggestion
description: The California Department of Social Services provides the following Expectant Parent Payment. 
```

File: policyengine_us/parameters/gov/states/ca/infant/age_limit.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+description: California defines infants as children below this age threshold.
```
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/ca/infant/age_limit.yaml:1]: the formula does `<=` though, and the test says 1 is eligible

File: policyengine_us/tests/policy/baseline/gov/local/ca/la/dss/infant_supplement/ca_la_infant_supplement.yaml [hunk 3]
```diff
@@ -0,0 +1,64 @@
+- name: Eligble household with one infant, not in group home
```
Comment (MaxGhenis) [inline policyengine_us/tests/policy/baseline/gov/local/ca/la/dss/infant_supplement/ca_la_infant_supplement.yaml:1]:
```suggestion
- name: Eligible household with one infant, not in group home
```

File: policyengine_us/variables/gov/local/ca/la/dss/expectant_parent_payment/ca_la_expectant_parent_payment.py [hunk 4]
```diff
@@ -0,0 +1,13 @@
+from policyengine_us.model_api import *
+
+
+class ca_la_expectant_parent_payment(Variable):
+    value_type = float
+    entity = Person
+    definition_period = MONTH
+    label = "Los Angeles County expectant parent payment"
+    defined_for = "ca_la_expectant_parent_payment_eligible"
+
```
Comment (MaxGhenis) [inline policyengine_us/variables/gov/local/ca/la/dss/expectant_parent_payment/ca_la_expectant_parent_payment.py:10]: right? or [] around?
```suggestion
    adds = "gov.local.ca.la.dss.expectant_parent_payment.amount"
```

File: policyengine_us/variables/gov/local/ca/la/dss/infant_supplement/ca_la_infant_supplement.py [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+        base_amount = where(
+            is_in_group_home, p.amount.group_home, p.amount.base
+        )
+        eligible_infants = add(
+            household, period, ["ca_la_infant_supplement_eligible_infant"]
+        )
+        eligible_person = person(
+            "ca_la_infant_supplement_eligible_person", period
+        )
+        eligble = household.any(eligible_person)
```
Comment (MaxGhenis) [inline policyengine_us/variables/gov/local/ca/la/dss/infant_supplement/ca_la_infant_supplement.py:26]:
```suggestion
        eligible = household.any(eligible_person)
```

File: policyengine_us/variables/gov/local/ca/la/dss/expectant_parent_payment/ca_la_expectant_parent_payment.py (hunk 4 above)
Comment (PavelMakarchuk) [reply policyengine_us/variables/gov/local/ca/la/dss/expectant_parent_payment/ca_la_expectant_parent_payment.py:10]: Forgot about the [] - works here

**Comment by PavelMakarchuk** [comment]
> please point to the WIC where relevant https://leginfo.legislature.ca.gov/faces/codes_displaySection.xhtml?lawCode=WIC&sectionNum=11465.
> 
> I think some of this is actually dependent on other policy parameters around foster care

If referring to the amounts - I am not sure which other program WIC refers to - I have added the legal code to the relevant sections

File: policyengine_us/variables/gov/local/ca/la/dss/infant_supplement/ca_la_infant_supplement_eligible_infant.py [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+class ca_la_infant_supplement_eligible_infant(Variable):
+    value_type = bool
+    entity = Person
+    definition_period = YEAR
+    label = "Eligible Infant for the Los Angeles County infant supplement"
+    defined_for = "in_la"
+
+    def formula(person, period, parameters):
+        p = parameters(period).gov.local.ca
+        age = person("age", period)
```
Comment (MaxGhenis) [inline policyengine_us/variables/gov/local/ca/la/dss/infant_supplement/ca_la_infant_supplement_eligible_infant.py:13]:
```suggestion
```

**Review by MaxGhenis — APPROVED** [review]

File: policyengine_us/variables/gov/local/ca/la/dss/infant_supplement/ca_la_infant_supplement_eligible_infant.py [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+
+class ca_la_infant_supplement_eligible_infant(Variable):
+    value_type = bool
+    entity = Person
+    definition_period = YEAR
+    label = "Eligible Infant for the Los Angeles County infant supplement"
+    defined_for = "in_la"
+
+    def formula(person, period, parameters):
+        p = parameters(period).gov.local.ca
```
Comment (MaxGhenis) [inline policyengine_us/variables/gov/local/ca/la/dss/infant_supplement/ca_la_infant_supplement_eligible_infant.py:12]:
```suggestion
```

**Pull request merged**